src/code/bank20.asm

```diff
@@ -2376,56 +2376,68 @@ jr_020_58EC:
 .jr_5901
     jr   jr_020_58B3                              ; $5901: $18 $B0
 
-ret_020_5903:
+.return:
     ret                                           ; $5903: $C9
 
 InventoryEntryPoint::
     ld   a, [wGameplaySubtype]                    ; $5904: $FA $96 $DB
     JP_TABLE                                      ; $5907
-._00 dw InventoryInitialHandler
-._01 dw InventoryMapFadeOutHandler
-._02 dw InventoryLoad1Handler
-._03 dw InventoryLoad2Handler
-._04 dw InventoryLoad3Handler
-._05 dw InventoryLoad4Handler
-._06 dw InventoryLoad5Handler
-._07 dw InventoryFadeInHandler
-._08 dw InventoryInteractiveHandler
-._09 dw InventoryStatusInHandler
-._0A dw InventoryStatusHandler
-._0B dw InventoryStatusOutHandler
-._0C dw InventoryFadeOutHandler
+._00 dw InventoryInitialHandler     ; GAMEPLAY_INVENTORY_INITIAL
+._01 dw InventoryMapFadeOutHandler  ; GAMEPLAY_INVENTORY_MAP_FADE_OUT 
+._02 dw InventoryLoad1Handler       ; GAMEPLAY_INVENTORY_DELAY1
+._03 dw InventoryLoad2Handler       ; GAMEPLAY_INVENTORY_DELAY1
+._04 dw InventoryLoad3Handler       ; GAMEPLAY_INVENTORY_DELAY1
+._05 dw InventoryLoad4Handler       ; GAMEPLAY_INVENTORY_DELAY1
+._06 dw InventoryLoad5Handler       ; GAMEPLAY_INVENTORY_DELAY1
+._07 dw InventoryFadeInHandler      ; GAMEPLAY_INVENTORY_FADE_IN
+._08 dw InventoryInteractiveHandler ; GAMEPLAY_INVENTORY_INTERACTIVE
+._09 dw InventoryStatusInHandler    ; GAMEPLAY_INVENTORY_STATUS_IN
+._0A dw InventoryStatusHandler      ; GAMEPLAY_INVENTORY_STATUS
+._0B dw InventoryStatusOutHandler   ; GAMEPLAY_INVENTORY_STATUS_OUT
+._0C dw InventoryFadeOutHandler     ; GAMEPLAY_INVENTORY_FADE_OUT
 
 InventoryInitialHandler::
     ldh  a, [hIsGBC]
     and  a
-    jr   z, jr_020_5940                           ; $5925: $28 $19
+    jr   z, .notGBC                               ; $5925: $28 $19
 
+    ; If GBC, store current background and OAM
+    ; palettes in WRAM bank 3
     ld   hl, wBGPal1                              ; $5927: $21 $10 $DC
-    ld   c, $80                                   ; $592A: $0E $80
+    ld   c, $80 ; counter, copy all 128 palettes  ; $592A: $0E $80
```

Comment:
It's not 128 palettes, it's 128 bytes of palette, which is 64 RGB15 entries, or 16 palette "rows?"